src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC, SetStateAction } from 'react';
+import { TodoFilter } from '../TodoFilter';
+import { Todo } from '../../types/Todo';
+import { FilterTodosBy } from '../../types/FilterTodosBy';
+
+type Props = {
+  todos: Todo[],
+  filterBy: string,
+  setFilterBy: (string: SetStateAction<FilterTodosBy>) => void,
+};
+
+export const Footer:FC<Props> = ({
+  todos,
+  filterBy,
+  setFilterBy,
+}) => {
+  const completedTodos = () => {
```

Comment:
Consider renaming this function to `hasCompletedTodos` as it returns a boolean value.